### DISCUSSION TITLE
Experience with Hotwire and Turbo?

### DISCUSSION THREAD

**Discussion by daya** [bensheldon/good_job#655] [General]
I wonder if anyone has tried GoodJob in place of Redis with Rails 7 with Hotwire [hotwire-rails](https://github.com/hotwired/hotwire-rails) gem.
 
If not, then what kind limitations one may run into e.g. performance, higher resource consumption? when using GoodJob from within Puma workers or when using GoodJob as standalone instance.

**Comment by bensheldon** [comment]
Thanks for opening this issue. I am _also_ curious about people's experience, specifically with Turbo which uses ActiveJob to render HTML. 

Speculatively it should work just fine. I think the thing I'd keep an eye on is table size and queue depth. 

GoodJob has always chosen job safety/resiliency over queue performance, which is maybe maybe maybe not the appropriate trade-off for Turbo 🤷‍♀️
